src/VisualStudio/Core/Def/Implementation/ProjectSystem/IVisualStudioWorkspaceHost2.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem
+{
+    // TODO: Find a better name for this
+    internal interface IVisualStudioWorkspaceHost2
+    {
+        void OnIsComplete(ProjectId projectId, bool succeeded);
```

Comment:
What does 'IsComplete' mean?  i don't know what this function is for, or what is signifies to implementors of IVisualStudioWorkspaceHost2

Also, why do we need a '2' version?  This is an internal interface, can we not just move this method to IVisualStudioWorkspaceHost?